tests/git_bisect_toaster.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# These variables are fully defined in runtests.sh
+# Just short descripstions appear here
+# what container to run. Usually crops/toaster-master
+export IMAGE=crops/toaster-master
```

Comment:
:(.

The container effects the env. Especially what django and pip vs pip3 versions we have...